third_party/patches/bazel-diff-only-just-non-external-rules.patch

```diff
@@ -0,0 +1,19 @@
+commit 30020d68649461fdace4cd7e91d100a059197a1d
+Author: Daniel Wagner-Hall <dwagnerhall@apple.com>
+Date:   Thu Apr 21 18:21:46 2022 +0100
+
+    Only return rules (and not external ones)
```

Comment:
is there an upstream PR for it yet? Ideally that PR number would go in the filename of this patch, so it's easier for us to track when it's merged and the patch can be dropped.

Reply:
There is not - I'll happily kick off a discussion async :)

Reply:
sounds good, I often just create the draft PR at least as a placeholder so I don't forget to start the process of converging again with upstream